libraries/c_sdk/standard/ble/include/iot_ble_mqtt_serialize.h

```diff
@@ -32,6 +33,8 @@
 #ifndef IOT_BLE_MQTT_SERIALIZE_H
 #define IOT_BLE_MQTT_SERIALIZE_H
 
+#define bool    int
```

Comment:
This seems like a dangerous change... It wouldn't work for compilers that have C99 support.

Reply:
bool won't work with c89 compilers 
so we need to fix all this file